src/views/system-security-plans/__tests__/SystemSecurityPlanExportOfferingsView.spec.ts

```
@@ -0,0 +1,296 @@
+  });
+
+  it('edits an offering and reflects the change in the list', async () => {
+    offeringsData.current = [makeOffering()];
+    putMock.mockResolvedValueOnce({
+      data: { data: makeOffering({ title: 'Renamed Offering' }) },
+    });
+
+    const wrapper = mountView();
+    await findButton(wrapper, 'Edit').trigger('click');
+    const inputs = wrapper.findAll('input');
+    await inputs[inputs.length - 1].setValue('Renamed Offering');
```

ESLint

[error] Use .at(-1) instead of [inputs.length - 1]

(e18e/prefer-array-at)